Gemfile

```diff
@@ -57,6 +59,7 @@ end
 
```

Comment:
![image](https://github.com/beteland123/blogapp/assets/93039359/009677ed-5a2f-4818-8d60-7ccdb2966c08)

- [ ] Kindly remove all the section descriptions to make your readme descriptive and clean 👍